src/models/exhibitor.model.ts

```diff
@@ -10,7 +10,10 @@ const exhibitorSchema: Schema = new Schema({
   picture: String,
   place: String,
   sectors: Array<String>,
-  interests: Array<String>,
+  interests: {
+    type: Schema.Types.ObjectId,
+    ref: 'Interests',
+  },
```

Comment:
Ça devrait être un array d'id ici donc ça :
```suggestion
  interests: [{
    type: Schema.Types.ObjectId,
    ref: 'Interests',
  }],
```